subsys/bluetooth/host/classic/Kconfig

```diff
@@ -505,6 +505,17 @@ config BT_AVRCP_CONTROLLER
 	help
 	  This option enables the AVRCP profile controller function
 
+config BT_AVRCP_VENDOR_RX_DATA_BUF_SIZE
+	int "AVRCP Vendor Dependent RX data buffer Size"
+	default 1024
+	range 1024 $(INT16_MAX)
```

Comment:
Suggest to change as `512 $(INT16_MAX)` because the `requestcontinue` is used when data length is bigger than `512`.